<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title_label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Create Task</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="parent_label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Parent Task:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="name_label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="name_textbox"/>
   </item>
   <item>
    <widget class="QLabel" name="description_label">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="description_textbox"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dates_laout">
     <item>
      <layout class="QVBoxLayout" name="start_date_layout">
       <item>
        <widget class="QLabel" name="start_date_label">
         <property name="text">
          <string>Start Date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="start_date_edit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="end_date_layout">
       <item>
        <widget class="QLabel" name="end_date_label">
         <property name="text">
          <string>End Date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="end_date_edit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="hours_label">
     <property name="text">
      <string>Estimated Hours</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="hours_spinbox"/>
   </item>
   <item>
    <widget class="QCheckBox" name="done_checkbox">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
